02-csv-vector-raster-plotting.Rmd

```diff
@@ -3,7 +3,7 @@ layout: post
 title: "Lesson 02: CSV to Shapefile in R"
 date:   2015-10-24
 authors: [Joseph Stachelek, Leah Wasser]
-contributors: []
+contributors: [Megan A. Jones]
```

Comment:
Megan, Let's add Sarah Newman as a contributor as well please. I am going to work on the authors script - we can talk more about this as well! Someone helped me automate populating authors! :)